Sources/Sentry/include/NSURLSessionTask+Sentry.m

```diff
@@ -0,0 +1,24 @@
+#import "NSURLSessionTask+Sentry.h"
+
+
+@implementation SentryNSURLSessionTask
+
++ (nullable NSString *)graphQLOperationNameFromTask:(NSURLSessionTask *_Nonnull)task
```

Comment:
`m`: It would be great if we could write this code in Swift. See https://github.com/getsentry/sentry-cocoa/blob/main/develop-docs/Swift-Usage.md for some guidance.